### ISSUE TITLE
Typescript: Missing comma in single line interface without semicolons

### ISSUE THREAD

**Issue by Rudeg** [prettier/prettier#1874]
Input:
```js
interface Test { one: string, two: any[] }
```

Output:
```js
interface Test { one: string two: any[] }
```

Should:
```js
interface Test { one: string; two: any[] }
```

https://prettier.github.io/prettier/#%7B%22content%22%3A%22interface%20Test%20%7B%20one%3A%20string%2C%20two%3A%20any%5B%5D%20%7D%5Cn%22%2C%22options%22%3A%7B%22printWidth%22%3A80%2C%22tabWidth%22%3A2%2C%22singleQuote%22%3Afalse%2C%22trailingComma%22%3A%22none%22%2C%22bracketSpacing%22%3Atrue%2C%22jsxBracketSameLine%22%3Afalse%2C%22parser%22%3A%22typescript%22%2C%22semi%22%3Afalse%2C%22useTabs%22%3Afalse%2C%22doc%22%3Afalse%7D%7D

**Comment by vjeux** [comment]
Good catch! cc @azz

**Comment by vjeux** [comment]
I just published 1.4.1 which fixes it!
